sdk/daml-script/daml3/Daml/Script/Internal/Questions/Submit/Error.daml

```diff
@@ -19,6 +19,7 @@ data UpgradeErrorType
   = ValidationFailed
   | DowngradeDropDefinedField
   | ViewMismatch
+  | ContractNotUpgradable
```

Comment:
FYI: very shortly, this data structure will change - ref: https://github.com/digital-asset/daml/pull/20355/files#diff-ae141f40e9e195f3a58ed9be65341a4b5564ad0bd2978c9327404c7e498bfadf

Reply:
General (forward) intent is to ensure we wire up various info fields in the error so that daml-script can reference them.

Reply:
As discussed offline, I'll aim to pick up the daml-script changes

Reply:
daml-script changes added on https://github.com/digital-asset/daml/pull/20355